src/components/calculator.js

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react';
+import calculate from '../logic/calculate';
+
+function Calculator() {
+  const [valueState, setValueState] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
+
+  const getButtonValue = (e) => {
+    const buttonName = e.target.innerHTML;
+    setValueState(calculate(valueState, buttonName));
+  };
+
+  return (
+    <div className='calculator'>
+      <div className='display'>{valueState.total}  { valueState.operation} {valueState.next}</div>
+      <div className='row'>
+        <button onClick={ getButtonValue }>AC</button>
+        <button onClick={ getButtonValue }>+/-</button>
+        <button onClick={ getButtonValue }>%</button>
+        <button className='yellow' onClick={ getButtonValue }>/</button>
```

Comment:
- [ ] Kindly fix this error in the division operation 
![image](https://github.com/sheezu900/Math-project/assets/24703171/a3212a65-d427-4f07-9f8b-fe2f9c042248)

- ### Hint:
You can fix it by changing the button text to `÷` instead of `/`